library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/StatelessSection.java

```diff
@@ -64,13 +20,25 @@ public StatelessSection(SectionParameters sectionParameters) {
             throw new IllegalArgumentException("Stateless section shouldn't have a loading state resource");
         }
 
+        if (sectionParameters.loadingViewWillBeProvided) {
```

Comment:
could you please add unit tests for these checks?

Reply:
Done. Added constructor_withXyzViewProvided_throwsException.